### ISSUE TITLE
Configure ResourceAuthorizationManager

### ISSUE THREAD

**Issue by totof3110** [michael-simons/wro4j-spring-boot-starter#4]
First off, thanks for the great Wro4j starter!

Our CSS has references to fonts, PNGs, etc. that are contained within WebJARS. Wro4j rewrites those request to something like:
```
http://localhost:8080/wro4j/styles.css?wroAPI=wroResources&id=classpath:META-INF/resources/webjars/font-awesome/4.6.2/fonts/fontawesome-webfont.woff2?v=4.6.2
```

Those requests gets caught in the default impl of `ResourceAuthorizationManager` (`DefaultResourceAuthorizationManager`) which rejects them as non-authorized.

Is there a way to configure that behavior?

Right now I'm doing the following:
```java
@Configuration
public class Wro4jConfiguration {
	@Autowired
	public void setWroManagerFactory(WroManagerFactory wroManagerFactory) {
		((BaseWroManagerFactory) wroManagerFactory).setResourceAuthorizationManager(authorizationManager());
	}

	private ResourceAuthorizationManager authorizationManager() {
		return new ResourceAuthorizationManager() {
			@Override
			public boolean isAuthorized(String uri) {
				return StringUtils.startsWithAny(uri, "classpath:public/", "classpath:META-INF/resources/webjars/");
			}
		};
	}
}
```
... but that feels hacky.

**Comment by michael-simons** [comment]
Hi, you're welcome :)

I'm not quite sure if that isn't a question that @alexo (wro4j creator) could answer better.

I have been using the starter with a lot of webjars, see https://github.com/EuregJUG-Maas-Rhine/site/blob/master/src/main/resources/wro.xml#L20 or https://github.com/michael-simons/biking2/blob/public/src/main/resources/wro.xml but not minimizing them.

Never saw that request pattern.

For me, the workaround feels hackish, too…

**Comment by totof3110** [comment]
I put up a small PR (https://github.com/michael-simons/wro4j-spring-boot-starter/pull/5) that allows me to configure a ResourceAuthorizationManager. With it, my config becomes:
```java
@Configuration
public class Wro4jConfiguration {
	@Bean
	public ResourceAuthorizationManager authorizationManager() {
		return new ResourceAuthorizationManager() {
			@Override
			public boolean isAuthorized(String uri) {
				return StringUtils.startsWithAny(uri, "classpath:public/", "classpath:META-INF/resources/webjars/");
			}
		};
	}
}
```

**Comment by michael-simons** [comment]
Totally. Regarding the content of the authorizationManager, I don't know. I'm gonna polish your pr a little bit, though.

**Comment by totof3110** [comment]
Actually some of these error were because my wro.xml had references like:
```
<css minimize="false">classpath:META-INF/resources/webjars/font-awesome/4.6.2/css/font-awesome.min.css</css>
```
if I change it to:
```
<css minimize="false">/webjars/font-awesome/4.6.2/css/font-awesome.min.css</css>
```
Then those references work.

However I have some other references like:
```
<css minimize="false">classpath:public/bower_components/bc-css-flags/dist/css/bc-css-flags.css</css>
```
which contain relative references (../) and those references get converted to `wroAPI=wroResources&id=classpath:`.

**Comment by michael-simons** [comment]
I have merged your PR, should be in central as 0.3.2 soon…

**Comment by totof3110** [comment]
Thanks!
